src/main/kotlin/blackjack/BlackJack.kt

```diff
@@ -1,36 +1,49 @@
 package blackjack
 
-import blackjack.entity.CardDrawer
-import blackjack.entity.Deck
-import blackjack.entity.Player
+import blackjack.entity.*
 import blackjack.ui.GetResult
 import blackjack.ui.Input
 
 class BlackJack {
     fun play() {
-        val names = Input.getNames()
+        val names: List<String> = Input.getNames()
         GetResult.informGameStart(names)
-        println()
 
-        val players: List<Player> = names.map { name -> Player(name, CardDrawer.drawInitialCards()) }
-        players.map { player: Player -> GetResult.printPlayerStatus(player) }
-        println()
+        val players = getPlayers(names)
```

Comment:
```suggestion
        val players = names.map { Player(it, CardDrawer.drawInitialCards()) }
```
이렇게 하면 한번에 한줄로 표현이 가능합니다 :) 

또한 딜러와 플레이어 객체를 생성할 때 `CardDrawer.drawInitialCards` 카드 2장을 뽑도록 되어있는데, 
생성자 `init` 키워드를 이용해서 생성 시에 각 딜러와 플레이어가 가지고 있는 카드에 2장씩 넣어주도록 하면 어떨까요?